test/recipes/30-test_evp_data/evppkey.txt

```diff
@@ -17257,6 +17257,35 @@ PeerKey=KAS-ECC-CDH_B-571_C24-Peer-PUBLIC
 Ctrl=ecdh_cofactor_mode:1
 SharedSecret=02da266a269bdc8d8b2a0c6bb5762f102fc801c8d5394a9271539136bd81d4b69cfbb7525cd0a983fb7f7e9deec583b8f8e574c6184b2d79831ec770649e484dc006fa35b0bffd0b
 
+# for cofactor-order points, CECDH should fail. Test that.
```

Comment:
did you mean ECDH, or what is CECDH ?

Reply:
> did you mean ECDH, or what is CECDH ?

I meant co-factor ECDH, but perhaps I just made up the CECDH acronym :) Technically it's ECC CDH. I'll fix it. (The `Ctrl` entry toggles this ECDH variant.)